tap_zoom/client.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, config, config_path):
         self.__refresh_token = None
         self.__use_jwt = False
         self.__expires_at = None
+        self.start_date = config.get('start_date')
```

Comment:
this means that the other streams don't have a concept of start_date, right? it'll always just be a full sync since there's no bookmarks other than sync_recordings

Reply:
also this is set by unused

Reply:
edit: nevermind im dumb

Reply:
Haha no worries, it's a bit confusing for sure since this tap does things in an _interesting_ way.